### PR TITLE
chore: consistently use .R[1,2] instead of _R[1,2]

### PR DESCRIPTION
**Pull request by a-frantz**

### CODE REVIEW

**Review by adthrasher — DISMISSED** [review]

File: tools/util.wdl [hunk 1]
```diff
@@ -780,7 +780,7 @@ task split_fastq {
             stream: true,
         }
         reads_per_file: "Number of reads to include in each output FASTQ file"
-        prefix: "Prefix for the FASTQ file. The extension `.fq.gz` will be added."
+        prefix: "Prefix for the FASTQ file. The extension `.fastq.gz` will be added."
```
Comment (a-frantz) [inline tools/util.wdl:783]: @adthrasher I'm realizing this description isn't exactly right. Should it be:
```suggestion
        prefix: "Prefix for the FASTQ file. The extension `<number>.fastq.gz` will be added."
```
The name is determined by the `split` tool, which is a kind of opaque

Comment (a-frantz) [reply tools/util.wdl:783]:
```suggestion
        prefix: "Prefix for the FASTQ files. The extension `<split number>.fastq.gz` will be added."
```

Comment (adthrasher) [reply tools/util.wdl:783]:
```suggestion
        prefix: "Prefix for the FASTQ file. A split index and the extension `.fastq.gz` will be added."?
```

Comment (a-frantz) [reply tools/util.wdl:783]: That reads to me like "a split index" is a separate file...
```suggestion
        prefix: "Prefix for the FASTQ files. The extension `.fastq.gz` (preceded by a split index) will be added."
```
?

Comment (adthrasher) [reply tools/util.wdl:783]: That works for me

**Review by adthrasher — APPROVED** [review]

**Pull request merged**